cns/configuration/env.go

```diff
@@ -12,12 +12,20 @@ const (
 	// EnvNodeIP is the IP of the node running this CNS binary
 	EnvNodeIP = "NODE_IP"
 	// LabelSwiftV2 is the Node label for Swift V2
-	LabelSwiftV2 = "kubernetes.azure.com/podnetwork-multi-tenancy"
+	LabelSwiftV2   = "kubernetes.azure.com/podnetwork-multi-tenancy-enabled"
```

Comment:
Do we need to call it as "kubernetes.azure.com/podnetwork-enabled" to check more secondary interfaces (SR IOV or multitenant)

Reply:
@thatmattlong @rbtr